namespaces/kubernetes.yml

```diff
@@ -11,34 +11,39 @@ namespace:
       enabled: true
     description: >
       The name of the pod
+      In case this logs carries a kubernetes event that was triggered by a pod only, it is overwritten by that pod's pod_name.
 
   - name: pod_id
     type: string
     norms:
       enabled: true
     description: >
       Kubernetes ID of the pod.
+      In case this logs carries a kubernetes event that was triggered by a pod only, it is removed.
```

Comment:
Is it removed because we don't have a `pod_id` value?  Or because it is not needed?